doc/getting_started/installation_win.rst

```diff
@@ -27,28 +27,58 @@ The simplest way to install a development environment on Windows is to use
 MSYS2, a modern UNIX environment for Windows. Follow the steps below to set it
 up:
 
-1. Download and install :program:`MSYS2`. Download the appropriate (32 or
+#. Download and install :program:`MSYS2`. Download the appropriate (32 or
    64-bit) MSYS2 installer from the `MSYS2 website`_ and execute it. On the
    final installation screen, check the "Run MSYS2 now." box to start up an
    MSYS2 shell when installation is complete.  Follow the rest of the
    installation instructions on the MSYS2 website to update the package
    database and core system packages.  You may be advised to "terminate MSYS2
    without returning to shell and check for updates again".  If so, simply
-   close the ``MSYS2 MSYS`` desktop app and run it again to complete the update.)
+   close the ``MSYS2 MSYS Shell`` desktop app and run it again to complete the update.)
 
-#. Launch the ``MSYS2 MSYS`` desktop app from your start menu (if it's not still open).
+#. Launch the ``MSYS2 MSYS Shell`` desktop app from your start menu (if it's not still open).
 
 #. If you're behind a corporate firewall, you'll likely need to specify a
    proxy to get access to internet resources::
 
       $ export http_proxy=http://proxy.mycompany.com:123
       $ export https_proxy=$http_proxy
 
-#. Install the dependencies required to build Zephyr:
+#. Install the dependencies required to build Zephyr::
 
    .. code-block:: console
 
-      $ pacman -S git make gcc diffutils ncurses-devel python2 python3
+      $ pacman -S git make gcc diffutils ncurses-devel python3
+
+#. Install pip and the required Python modules::
+
+      $ curl -O 'https://bootstrap.pypa.io/get-pip.py'
+      $ ./get-pip.py
+      $ rm get-pip.py
+
+      $ pip install pyaml
```

Comment:
At least on Arch, that's where pacman puts system-wide Python packages (also see https://docs.python.org/3/library/site.html). Can't say for sure, but it seems like that might conflict with the package manager on MSYS2 as well.

Does MSYS2's pacman have python-yaml available? If so, could that be used instead to avoid the pip installation step? Otherwise my guess is that it's a good idea to add --user to the pip install line, just for cleanliness.